x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts

```diff
@@ -359,6 +359,16 @@ export const saveTimeline = () => {
   });
 };
 
+export const confirmLeaveUnsavedTimeline = () => {
+  cy.get(DELETION_CONFIRMATION).click();
+};
+
+export const deleteTimeline = () => {
+  cy.get(TIMELINE_COLLAPSED_ITEMS_BTN).click();
+  cy.get(DELETE_TIMELINE_BTN).click();
+  cy.get(DELETION_CONFIRMATION).click();
+};
+
```

Comment:
I'm getting errors where these constants are not defined

Reply:
Rebase error, will address, thanks!